src/components/executive-education-2u-error/ExecutiveEducation2UErrorPage.test.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { getAuthenticatedUser } from '@edx/frontend-platform/auth';
+import { getConfig } from '@edx/frontend-platform/config';
+import '@testing-library/jest-dom/extend-expect';
+
+import ExecutiveEducation2UErrorPage from './ExecutiveEducation2UErrorPage';
+
+jest.mock('@edx/frontend-platform/auth');
+jest.mock('@edx/frontend-platform/config');
+jest.mock('@edx/frontend-component-footer', () => () => <div data-testid="site-footer" />);
+
+getAuthenticatedUser.mockReturnValue({
+  id: 1,
+  profileImage: {
+    imageUrlMedium: 'htts://img.url',
+  },
+});
+
+getConfig.mockReturnValue({
+  MARKETING_SITE_BASE_URL: 'https://marketing.url',
+  LEARNER_SUPPORT_URL: 'https://support.url',
+  LOGOUT_URL: 'https://logout.url',
+});
+
+describe('ExecutiveEducation2UErrorPage', () => {
+  test('properly renders page layout with no parameters', async () => {
+    const tree = renderer
+      .create((
+        <ExecutiveEducation2UErrorPage />
+      ))
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  test('properly renders page layout with generic failure_reason (no offer available) parameter and url redirect', async () => {
```

Comment:
Looking at the generated snapshot, it seems like the failure reason is all "Error occurred while processing your request". It seems like there might be something missing?

Reply:
Consolidated component into 2UPage